### ISSUE TITLE
Improve splash screen for dark theme

### ISSUE THREAD

**Issue by Ammako** [mollyim/mollyim-android#142]
### Is there an existing request for this?

- [X] I have searched the existing issues

### Feature description

It would be nice if Molly could have separate splash screens for light mode and dark mode users, like Signal has.

<details>
  <summary>Pics:</summary>
  
  ![Screenshot_20221027-145910](https://user-images.githubusercontent.com/108308111/198376052-cf1670aa-9dc2-48ca-8d8a-90e77c9bc825.png)
  
  ![Screenshot_20221027-145713](https://user-images.githubusercontent.com/108308111/198376068-ccf8528d-55a8-4565-86fc-92ec374d4527.png)
  
</details>

Auto-generated by Android 12+ as per https://github.com/signalapp/Signal-Android/issues/12076#issuecomment-1081203836

Hopefully it's possible to keep the purple splash screen for light mode users at least, because your purple splash is quite nice. It's just when in dark mode, it would be nice if a darker splash could be used. Maybe a darker shade of purple? NewPipe does that, lighter red in light mode and darker red in dark mode.

**Comment by valldrac** [comment]
You're right. The current splash screen is auto-generated from the launcher icon, and the launcher icon has the same background color for both themes.

The correct way to fix this is to make an adaptive icon with different background colors for each themes, and set `windowSplashScreenBackground` accordingly. The icon should follow the requirements defined in: https://developer.android.com/develop/ui/views/launch/splash-screen#splash-screen-animate-reqs

If you want to contribute, please make the icon and submit the pull-request. I will merge it.
